js/__loader.js

```diff
@@ -290,13 +290,17 @@
   const stream = loader.require('stream');
   class StdoutStream extends stream.Writable {
     _write(chunk, encoding, callback) {
-      __SYSCALL.log(String(chunk));
+      var out = String(chunk);
+      if (out.substring(out.length-1) === '\n') out = out.substring(0, out.length-1);
```

Comment:
It is, but if you remove the newline from `__SYSCALL.log`, the output at startup is output on the same line.